strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.9__Repository_group_repositories.xml

```diff
@@ -0,0 +1,15 @@
+<databaseChangeLog xmlns:o="http://www.unbroken-dome.org/schema/liquibase-orientdb"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.2.xsd">
+
+    <changeSet id="v1.0.0.9" author="przemyslaw.fusik@gmail.com">
+
+        <o:dropProperty propertyName="groupRepositories" className="Repository"/>
+
+        <o:createProperty name="groupRepositories" type="embeddedlist" linkedType="string" className="Repository"/>
```

Comment:
Is there any way to move the data between properties, @sbespalov ?

Reply:
I don't know 
this needed to be investigated, also seems that there is and issue with such schema manipulation when we drop one property and create another. @steve-todorov reported and issue [SB-1112](https://youtrack.carlspring.org/issue/SB-1112) about this which I trying to resolve. 
So can you please double check that this migration script works correctly?

Reply:
I've built a `strongbox-distribution` from this PR and used it on our existing `strongbox-snapshot` database. Everything seems to be working so far and I didn't encounter any issues.

Reply:
Thank you, @steve-todorov for tests, unfortunately, it needs to be further improved.